examples/kfctl_istio_dex.v1.2.0.yaml

```diff
@@ -0,0 +1,96 @@
+apiVersion: kfdef.apps.kubeflow.org/v1
+kind: KfDef
+metadata:
+  name: kubeflow-kubermatic
+  namespace: kubeflow
+spec:
+  applications:
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: namespaces/base
+    name: namespaces
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: application/v3
+    name: application
+  - kustomizeConfig:
+      parameters:
+      - name: serviceType
+        value: LoadBalancer
+      repoRef:
+        name: flowmatic
+        path: manifests/istio/istio-1-3-1-stack/overlays/ingressgateway
```

Comment:
Here we just point to our new base.

Reply:
I'm, just thinking, since this is our new base, should't we rather put it into `manifests/istio/istio-1-3-1-stack` instead of `manifests/istio/istio-1-3-1-stack/overlays/ingressgateway` ?

Reply:
yeah, you are definitely right, now it is not an overlay anymore, we can move it to `manifests/istio/istio-1-3-1-stack`